apps/dav/lib/CalDAV/CalDavBackend.php

```diff
@@ -2987,7 +2988,15 @@ public function getDenormalizedData(string $calendarData): array {
 					$lastOccurrence = $firstOccurrence;
 				}
 			} else {
-				$it = new EventIterator($vEvents);
+				try {
+					$it = new EventIterator($vEvents);
+				} catch (NoInstancesException $e) {
+					$this->logger->error('Caught no instance exception for calendar data. This usually indicates invalid calendar data.', [
```

Comment:
```suggestion
					$this->logger->debug('Caught no instance exception for calendar data. This usually indicates invalid calendar data.', [
```

Error will show up on prod but admins can't do anything about it really

Debug level is sufficient for development and debugging :+1:

Reply:
Done